interface/src/Application.cpp

```diff
@@ -5534,6 +5539,21 @@ void Application::loadSettings() {
     }
 
     getMyAvatar()->loadData();
+
+
+    auto bucketEnum = QMetaEnum::fromType<ExternalResource::Bucket>();
+    auto er = ExternalResource::getInstance();
```

Comment:
```suggestion
    auto externalResource = ExternalResource::getInstance();
```
Coding Standard 1.2.12